src/proto/h1/role.rs

```diff
@@ -1011,6 +1127,30 @@ impl Http1Transaction for Client {
         }
     }
 
+    #[cfg_attr(feature = "layers",
+        instrument( level = "trace",
+                    skip(msg),
```

Comment:
probably also don't want to record `dst` here...
```suggestion
                    skip(msg, dst),
```